.github/workflows/changelog-checker.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow checks that there is either a 'pr/no-changelog' label applied to a PR
+# or there is a changelog/<pr number>.txt file associated with a PR for a changelog entry
+
+name: Check Changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+    # Runs on PRs to master
+    branches:
+      - master/*
```

Comment:
Why limit to master, don't we want them on releases too?

Reply:
Theoretically that should already be caught by the master check?

Reply:
But also why not, I guess.

Reply:
Also, I think that trailing `/*` shouldn't be there for master.